### PR TITLE
fix(fix production slider): let's see if upgrading rc-slider would fix production build

### PR DESCRIPTION
**Pull request by esonhey**
HFRONT-901

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,7 +61,6 @@
     "@storybook/test-runner": "^0.5.0",
     "@types/node": "^12.12.38",
     "@types/ramda": "^0.28.1",
-    "@types/rc-slider": "^8.6.6",
```
Comment (esonhey) [inline package.json:64]: no need extra type library

File: src/components/picker/handler/index.tsx [hunk 2]
```diff
@@ -1,5 +1,4 @@
 import React, { useContext, useEffect } from 'react';
-import { Handle } from 'rc-slider';
```
Comment (esonhey) [inline src/components/picker/handler/index.tsx:2]: Handle component is being removed from 'rc-slider'

File: src/components/picker/handler/index.tsx [hunk 3]
```diff
@@ -26,9 +25,9 @@ const PickerHandler = React.forwardRef((props: any) => {
       index={index}
       key={index}
     >
-      <Handle value={value} {...restProps}>
+      <div value={value} {...restProps}>
```
Comment (esonhey) [inline src/components/picker/handler/index.tsx:28]: works fine with div

File: src/components/picker/multi-range-slider/index.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React, { FC, useCallback, useContext, useMemo } from 'react';
 import { Box } from 'rebass';
-import { Range } from 'rc-slider';
+import RcSclider from 'rc-slider';
```
Comment (esonhey) [inline src/components/picker/multi-range-slider/index.tsx:3]: single default export component which accecpts `range` prop instead to create RangeSliders

**Review by snqb — APPROVED** [review]
looks good

**Pull request merged**